CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [Added Batch transactions support](https://github.com/multiversx/mx-sdk-dapp/pull/722)]
```

Comment:
this will generate a new build, needs version above
also run `npm version 2.11.0`